docs/content/philosophy.md

```diff
@@ -78,4 +78,77 @@ dynamically into OPA remotely via APIs or through the local filesystem.
 
 OPA is a full-featured policy engine that offloads policy decisions from your
 service. You can think of it as a concierge for your service who can answer
-detailed questions on behalf of your users to meet their specific needs.
\ No newline at end of file
+detailed questions on behalf of your users to meet their specific needs.
+
+## How Does OPA Work? {#how-does-opa-work}
+
+> 📖 This section assumes you are familiar with the basics of OPA. We recommend
+> going through the [Introduction](..) before reading this section.
```

Comment:
The requirement for going through the intro doesn't seem like the right thing for this page.  If you're reading this page you  you don't know what OPA is (section 1) or why you might use it (section 2).

Reply:
Agreed. I've amended the PR and I'm happier with the new version.